File: HackerRank Problem/Python/aVeryBigSum.py
#!/bin/python3

import os


# Complete the aVeryBigSum function below.
def aVeryBigSum(ar):
    b = 0
    for i in range(0, ar_count):
        b = b+ar[i]
    return b


if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')
    ar_count = int(input())
    ar = list(map(int, input().rstrip().split()))
    result = aVeryBigSum(ar)
    fptr.write(str(result) + '\n')
    fptr.close()

File: HackerRank Problem/Ruby/AVeryBigSum.rb
#!/bin/ruby

require 'json'
require 'stringio'

# Complete the aVeryBigSum function below.
def aVeryBigSum(ar)
    ar.reduce(&:+)
end

fptr = File.open(ENV['OUTPUT_PATH'], 'w')

ar_count = gets.to_i

ar = gets.rstrip.split(' ').map(&:to_i)

result = aVeryBigSum ar

fptr.write result
fptr.write "\n"

fptr.close()
